tests/e2e/e2e_setup_test.go

```diff
@@ -112,7 +114,15 @@ func (s *IntegrationTestSuite) SetupSuite() {
 	s.T().Log("setting up e2e integration test suite...")
 
 	// Setup bech32 prefix
-	cmd.InitSDKConfig()
+	if !s.initialized {
+		cmd.InitSDKConfig()
+
+		// note: the config gets sealed on init, so runnig this twice will throw!
+		s.initialized = true
+	}
```

Comment:
SetupSuite()` is called multiple times because of some specific cases that require a restart of the chain (like the gov chain software proposal). You can move the call to `cmd.InitSDKConfig()` to `TestIntegrationSuite()` which is guaranteed to be called once. No need for additionnal field like `initialized`.